kotlin_big/build.gradle

```diff
@@ -88,21 +88,34 @@ task downloadKotlinSources(type: Download) {
 }
 
 task extractKotlinSources(type: Sync, dependsOn: downloadKotlinSources) {
-  enabled = findProperty("kotlinLocalRoot") == null
-  from zipTree(kotlinSourcesZip)
-  include "*/libraries/**"
-  include "*/core/**"
+  String pathPrefix = ""
+
+  if (project.findProperty("kotlinLocalRoot") != null) {
+      from project.property("kotlinLocalRoot")
+      exclude "**/build"
+      exclude "**/node_modules"
+  } else {
+      from zipTree(kotlinSourcesZip)
+      pathPrefix = "*/"
+
+      eachFile { FileCopyDetails file ->
+          //assume all files in the folder
+          file.path = file.path.split("/", 2)[1]
+      }
+  }
+
+  include "${pathPrefix}libraries/stdlib/**"
+  include "${pathPrefix}libraries/kotlin.test/**"
+  include "${pathPrefix}core/builtins/**"
+  include "${pathPrefix}core/reflection.jvm/**"
+
+  exclude "${pathPrefix}core/builtins/src/kotlin/reflect/"
```

Comment:
This `exclude` seems to be unreliable. I've noticed the task remains up-to-date even when exclude spec changes. So perhaps it's worth to invoke `clean` task before `build`.

Reply:
I'm ok to use `Copy` task instead, but it will be slower. We do have clean checkout on the CI, so every build will stary from an empty dir

Reply:
>  We do have clean checkout
Currently we don't

Reply:
Let's fix it on TeamCity. Ping me if you need help!

Reply:
Done. I've added `clean` task to kotlin-dokka-stdlib project build.